data/hades/tags/worldgen/biome/any_overworld_biome.json

```diff
@@ -1,8 +1,8 @@
 {
   "values": [
```

Comment:
I don't understand those changes.
We need to know if those tags are defining the generation of a pilgrim's path temple

Reply:
Also, I am not sure you added those tag files to the version control system.

Reply:
Yep, I checked and some files are missing, and the naming convention is somewhat incorrect.

Reply:
Whoops, sorry about that. My "search and replace" tool must have disobeyed me and remove those parts of tags names, while I haven't even change those in path.

Reply:
It should be correct now, that I restored them back